### DISCUSSION TITLE
Is it one event?

### DISCUSSION THREAD

**Discussion by lenkazejskova** [HGVSnomenclature/hgvs-nomenclature#197] [Q&A]
Hallo,
I would like to ask you if you think that the complex change in WT1 gene is the only one event and should be described as indel (using hgvs nomenclature) or more events. The patient has three mutations (I think)  in probably different clones, two separable from the others and one is more complex, see IGV picture.
![wt1](https://github.com/user-attachments/assets/2ee747b6-a576-4d1e-8d4e-5cb26073183c)

Please, what do you think?

Thank you very much
Lenka

**Comment by jfjlaros** [comment]
It looks to me that there are three alleles present in this picture:

- One containing a deletion, an insertion of seven nucleotides, two SNVs and a deletion (e.g., reads one to three).
- One containing only one insertion (e.g., reads four to six).
- One containing only one insertion of five nucleotides (e.g., read sixteen). Given its low frequency (from what I can see), this could be a false positive.

You can write this down as three alleles enclosed in `[]` and separated by a `;`. E.g.,

    NC_000011.x:g.[110_114del;123_124insXXXXXXX;127X>C;131X>A;140_141insX];[100_101insX];[135_136insXXXXX]

Of course, all coordinates are made up and `x` and `X` are placeholders.

There are no rules against writing the first allele as a large delins. E.g.,

        NC_000011.x:g.[110_140delinsXXX...XXX];[100_101insX];[135_136insXXXXX]

Does this help?

**Comment by lenkazejskova** [comment]
Dear Jeroen,
yes, it helped, thank you very much. We usually discuss if more complex change has been one event during mutation process or not. Concerning this, sometimes HGVS criteria about spannning nucleotides are the object of long discussion 🙂
 "two variants that are separated by fewer than two intervening nucleotides (that is, not including the variants themselves) should be described as a single "delins" variant."

I am totally for to describe the first one discussed change as delins and not as more separated mutations 🙂

Thanks a lot again
Best regards
Lenka
________________________________
Od: Jeroen F.J. Laros ***@***.***>
Odesláno: středa 6. listopadu 2024 18:20
Komu: HGVSnomenclature/hgvs-nomenclature ***@***.***>
Kopie: ŽEJŠKOVÁ Lenka Mgr.Ph.D. ***@***.***>; Author ***@***.***>
Předmět: Re: [HGVSnomenclature/hgvs-nomenclature] Is it one event? (Discussion #197)


It looks to me that there are three alleles present in this picture:

  *   One containing a deletion, an insertion of seven nucleotides, two SNVs and a deletion (e.g., reads one to three).
  *   One containing only one insertion (e.g., reads four to six).
  *   One containing only one insertion of five nucleotides (e.g., read sixteen). Given its low frequency (from what I can see), this could be a false positive.

You can write this down as three alleles enclosed in [] and separated by a ;. E.g.,

NC_000011.x:g.[110_114del;123_124insXXXXXXX;127X>C;131X>A;140_141insX];[100_101insX];[135_136insXXXXX]


Of course, all coordinates are made up and x and X are placeholders.

There are no rules against writing the first allele as a large delins. E.g.,

    NC_000011.x:g.[110_140delinsXXX...XXX];[100_101insX];[135_136insXXXXX]


Does this help?
